cht-llama-cpp/docker/cpu/Dockerfile

```diff
@@ -1,22 +1,17 @@
 FROM python:3.10-slim-bullseye
-
-ARG MODEL_ID
-
-RUN apt update && apt install -y libopenblas-dev ninja-build build-essential wget
-RUN python -m pip install --upgrade pip pytest cmake scikit-build setuptools
```

Comment:
is this required? seen it in a few places

Reply:
Maybe. But we will deprecate Vicuna and GPT4All, so no big deal.